fiberplane-models/src/front_matter_schemas.rs

```diff
@@ -0,0 +1,316 @@
+use crate::timestamps::Timestamp;
+use base64uuid::Base64Uuid;
+#[cfg(feature = "fp-bindgen")]
+use fp_bindgen::prelude::Serializable;
+use ordered_float::OrderedFloat;
+use serde::{Deserialize, Serialize};
+use strum_macros::Display;
+use typed_builder::TypedBuilder;
+
+/// A floating-point number that can be ordered and compared using Eq.
+///  
+/// It is not compliant to IEEE standard, and NaN is considered greater than
+/// everything and equal to itself.
+///
+/// Also, this type is serializable using fp-bindgen, transparently to the underlying
+/// f64 primitive.
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize, Default)]
+#[repr(transparent)]
+pub struct SerializableEqFloat(OrderedFloat<f64>);
+
+#[cfg(feature = "fp-bindgen")]
+impl Serializable for SerializableEqFloat {
+    fn ident() -> fp_bindgen::types::TypeIdent {
+        fp_bindgen::types::TypeIdent::from("f64")
+    }
+
+    fn ty() -> fp_bindgen::types::Type {
+        fp_bindgen::types::Type::Primitive(fp_bindgen::primitives::Primitive::F64)
```

Comment:
I think this *should* help so you don't need the extra patch in the FiberKit bindings:

```suggestion
        fp_bindgen::types::Type::Custom(fp_bindgen::types::CustomType {
            fp_bindgen::types::TypeIdent::from("SerializableEqFloat"),
            rs_ty: "f64".to_owned(),
            ts_ty: "number".to_owned(),
            ..Default::default()
        })
```

I'll admit it would've been nice if `fp-bindgen` auto-detected that you're mapping to a primitive and would've caught that for you :)

Reply:
I tried but I don’t think it changed anything (the patch applied cleanly the same way), maybe it was a caching issue I’ll keep an eye on it